examples/distributed_tree/distributed_knn.cpp

```diff
@@ -0,0 +1,104 @@
+/****************************************************************************
+ * Copyright (c) 2017-2022 by the ArborX authors                            *
```

Comment:
Copyright date is wrong
(Don't care too much about it)

Reply:
Don't care.